data/test/macros/unit_setup.cfg

```diff
@@ -32,17 +32,15 @@ no#endarg
 #arg CUMULATIVE
 no#endarg
 
-    [heals]
+    [{ABIL}]
         id = {ID}
-        name = _"test-heals_{ID}"
+        name = _"test-{ABIL}_{ID}"
         affect_self = {SELF}
         affect_allies = {ALLIES}
         affect_enemies = {ENEMIES}
         poison = {POISON}
```

Comment:
Who really knows - add/sub/multiply/divide aren't currently documented as being used for heals or regenerate either, but they all work. The only reason they aren't in the macro is that `sub` takes priority over `add`, so when you have both it (annoyingly) only uses `sub`.

Reply:
For heals and regenerate, that seems to be a documentation problem.

Will there be another bunch of tests coming for other abilities? I guess the question is, is this macro going to get used for the other abilities, and will it work or fail for them? (Yes, I could test that myself, but as you're asking whether there are more comments to come, I'm commenting now.)

Reply:
The first PR added around 30 files with `TEST_HEALS`. This PR adds ~30 new files, but also touches ~30 old files to rename it to `TEST_ABILITY`.

My worry is that the next PR will add ~30 files, but also touch ~60 old files to rename `TEST_ABILITY` to `TEST_HEALS_OR_REGEN`.

Reply:
I'm not sure why I'd rename it to that. I don't think it's much of a problem that I might need to go back and edit previous tests though.

Reply:
I'm expecting the next PR to rename it because the macro currently called `TEST_ABILITY` only works for `heals` or `regeneration`. Try `leadership`, and the schema will catch it:

```
error validation: Invalid key 'poison=' in tag [leadership]
at test/macros/unit_setup.cfg:28
    included from test/_main.cfg:30
    included from data/_main.cfg:69
```

Reply:
More likely I'd add it where necessary to `{OTHER}`. Schema saying it's wrong doesn't necessarily mean it won't work though.

Reply:
If the tests will change to move it to `{OTHER}`, then I think that should happen in this current PR.

Reply:
I don't know if it will or not - I haven't added those tests yet. My approach to this is that given all the debates about how abilities do work, don't work, or should work is that it doesn't appear that anyone really knows how abilities actually work, so I'm not assuming anything until I can add a test confirming it. So I will not be making this change during this PR.